strongbox-web-core/src/main/java/org/carlspring/strongbox/security/vote/FilterAccessDecisionManager.java

```diff
@@ -1,30 +0,0 @@
-package org.carlspring.strongbox.security.vote;
```

Comment:
I think yes, thanks to `MethodAccessDecisionManager` registered in SecurityConfig, but correct me if I'm wrong.